### PR TITLE
tailwind

### PR DESCRIPTION
**Pull request by pondohoo**
i'm trying to point them in the right direction on what to google and where to look at the tailwind docs without holding their hand through everything

### CODE REVIEW

File: docs/tailwind/colors.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+![Alt text](join-us-button-color.png)
+
+
+## Gradient colors
+
+> 💡  For more color gradient variations see the [Tailwind docs](https://tailwindcss.com/docs/gradient-color-stops)
+
+To add a left-to-right background gradient, apply the `bg-gradient-to-r` class, pick the left color with `from-{color}` and the right color with `to-{color}`.
```
Comment (SafeDuck) [inline docs/tailwind/colors.md:38]: I would also add one for how to do gradient on text: https://design2tailwind.com/blog/tailwindcss-gradient-text/

File: docs/tailwind/colors.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Solid colors
+
+The color style names are prefaced by the name of the website (e.g. `acm-blue-100`, `hackathon-red`, etc.).
+
+You can directly attach these color styles to Tailwind utility classes.
+
+In the R-Tools repo, for example, you could color the background of a container teal with the `bg-rtools-teal-100` class
+
+```html
+<button className="bg-rtools-teal-100 px-12 py-1 border border-solid border-black rounded-full">
```
Comment (SafeDuck) [inline docs/tailwind/colors.md]: What do you think about also putting something like `text-rtools-black` to how colors can be applied to other things too?

File: docs/tailwind/flexbox.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+<div className="flex items-center justify-center">
+  <div>child container</div>
+</div>
+```
+
+The `flex` class applies flexbox to the container. Flexbox allows containers to stretch and change order and orientation of their child containers. See [this link](https://tailwindcss.com/docs/flex) for a visualization of flexbox in Tailwind. The other two classes **only work when flex is applied**. 
+
+`items-center` centers the child div within the parent div **vertically**.
+
+  `justify-center` centers the child div within the parent div **horizontally**
```
Comment (SafeDuck) [inline docs/tailwind/flexbox.md]: I think this is a little vague and doesn't paint the whole picture:
`items-center` centers **vertically** only when there is a `flex-row`, and then centers **horizontally** when there is a `flex-col`
On the other side:
 `justify-center` centers **horizontally** only when there is a `flex-row`, and then centers **vertically** when there is a `flex-col`
In short, `items-center` is used to center items along the cross-axis and `justify-center` is used to center items along the main axis.
The main axis is row if it's `flex-row` and it's col when it's `flex-col.` 
The cross axis is row is it's `flex-col` and it's col when it's `flex-row`

**Review by SafeDuck — CHANGES_REQUESTED** [review]

File: docs/tailwind/flexbox.md (hunk 3 above)
Comment (pondohoo) [reply docs/tailwind/flexbox.md]: elaborated on it

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**